frontend/src/components/Score.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import "./Score.scss";
+
+class Score extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { score: 0 };
+    this.scoreUp();
+  }
+
+  scoreUp() {
+    this.setState({ score: this.state.score + 1 });
+    setTimeout(this.scoreUp.bind(this), 250);
```

Comment:
replaces the bind function with arrow function

Reply:
done